### ISSUE TITLE
Host

### ISSUE THREAD

**Issue by barrett092** [byt3bl33d3r/DeathStar#21]
Any reason why my host in Empire won't change to the Local IP address? Instead it just stays at 127.0.0.1

**Comment by barrett092** [comment]
When you launch the DeathStar listener that is..

**Comment by InfiniteSuns** [comment]
Same problem here, I've figured out a bypass, which works for me:
1. Run Empire with REST API, don't run DeathStar yet;
2. Configure and run regular Empire http listener with "Host" and "Port" set to your local IP and 8443;
3. Kill this http listener and start DeathStar, now it'll bind to your local IP, not 127.0.1.1.

**Comment by barrett092** [comment]
Thanks for the workaround!

**Comment by byt3bl33d3r** [comment]
@barrett092 @InfiniteSuns That usually happens when Empire fails to detect your interface. You can use any listener with DeathStar (not just the one it creates), it only creates it on runtime for convinience sake. 

TL;DR
Killing the listener, setting the correct IP and restarting it again will fix it.
